http-body-util/src/stream.rs

```diff
@@ -26,24 +25,22 @@ impl<S> StreamBody<S> {
 
 impl<S, D, E> Body for StreamBody<S>
 where
+    // === REVIEW: should this be a Stream of Frames, or Data? ===
```

Comment:
Wanted to flag this for others: technically a `Stream` of `Frame<D>` is more flexible, but also, I don't expect people will usually have a stream of frames, just of bufs...

Reply:
Does it really harm if we have a Stream of Frame vs D? I would imagine for tonic we convert a `Stream<Item = Result<Message, Status>>` that then gets converted into data frames and trailers for the status etc.

Reply:
I'm not sure which way you're advocating for.